src/Microsoft.AspNetCore.Testing/Microsoft.AspNetCore.Testing.csproj

```diff
@@ -4,14 +4,17 @@
 
   <PropertyGroup>
     <Description>Various helpers for writing tests that use ASP.NET Core.</Description>
-    <TargetFrameworks>netstandard2.0;net46</TargetFrameworks>
-    <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
```

Comment:
It's .NET Core 2.0 I'm concerned about. I don't know how PackageTargetFallback rolls into the compatibility selection.